pandas/tests/reshape/merge/test_multi.py

```diff
@@ -0,0 +1,623 @@
+# pylint: disable=E1103
+
+from collections import OrderedDict
+
+import numpy as np
+import pytest
+from numpy import nan
+from numpy.random import randn
+
+import pandas as pd
+import pandas.util.testing as tm
+from pandas import (DataFrame, Index, MultiIndex, Series)
+from pandas.compat import lzip
+from pandas.core.reshape.concat import concat
+from pandas.core.reshape.merge import merge
+
+
+@pytest.fixture
+def left():
+    # a little relevant example with NAs
```

Comment:
Make a docstring so that it shows up in `pytest --fixtures`. Maybe a bit more descriptive about the return type.

Reply:
Could something like that work?
"""left dataframe (not multi-indexed) for multi-index join tests"""